### DISCUSSION TITLE
Photos of print & builds

### DISCUSSION THREAD

**Discussion by OrpheeGT** [hrvach/deskhop#46] [General]
Hello !

I thought it would be fun to share our builds (like you can find on thingiverse) :)

So here is mine :p

![image](https://github.com/hrvach/deskhop/assets/8038551/d2fcdcc1-b7b1-47c9-9b4d-159ff414c3d1)

![image](https://github.com/hrvach/deskhop/assets/8038551/0d6c936e-8926-41d4-919f-45297acac067)

![image](https://github.com/hrvach/deskhop/assets/8038551/e694297e-86fb-459e-8637-762faf42ce25)

**Comment by hrvach** [comment]
Nice job!

**Comment by chris-010** [comment]
Here is my first batch of rev 1.1

![image](https://github.com/hrvach/deskhop/assets/10660568/6ace5c3e-483c-4a5e-bdb9-78cd52372fb4)

**Comment by jalmeroth** [comment]
Here's my collection 🌈

![image](https://github.com/user-attachments/assets/8194f96d-815a-449f-b93f-885ff9d95792)

Orange: Pico2 Dev-Board
Yellow: Pico Production
Green: Pico Dev-Board
